using-timescaledb/continuous-aggregates.md

```diff
@@ -139,19 +139,67 @@ PostgreSQL. In addition, TimescaleDB continuous aggregates does not
 currently support the `FILTER` clause even though it is possible to
 parallelize but we might add support for this in a future version.
 
+#### Creating a continuous aggregate without refreshing it
+
+By default, a continuous aggregate is refreshed when created, which
+means that it is immediately populated with data. Since this can be a
+lengthy process a notice will be printed informing you that the
+materialization is in progress.
+
+If you have a lot of data, you might want to disable this using the
+`WITH NO DATA` option to `CREATE MATERIALIZED VIEW`.
+
+```sql
+CREATE MATERIALIZED VIEW conditions_summary_daily
+WITH (timescaledb.continuous) AS
+SELECT device,
+       time_bucket(INTERVAL '1 day', time) AS bucket,
+       AVG(temperature),
+       MAX(temperature),
+       MIN(temperature)
+FROM conditions
+GROUP BY device, bucket
+WITH NO DATA;
+```
+
+#### Add a continuous aggregate policy
+
+You can refresh the continuous aggregate view manually as above, but
+it is much more convenient to add a policy that takes care of the
+refresh using
+[`add_continuous_aggregate_policy`][add-continuous-aggregate-policy],
+for example:
+
+```sql
+SELECT add_continuous_aggregate_policy('device_summary',
+	start_interval => NULL,
+	end_interval => INTERVAL '2 h',
+	schedule_interval => INTERVAL '2 h');
+```
+
+The policy will refresh a sliding window of the continuous aggregate
+that is given by a start and end interval, which are relative to
+[`CLOCK_TIMESTAMP`][clock_timestamp]. In the same was as for the
+`refresh_continuous_aggregate` function, providing `NULL` to either of
+the intervals will make it open-ended.
+
+The `schedule_interval` tells how often the policy job should run, so
+the most recent value in the continuous aggregate will be in the range
+`[end_interval, end_interval+schedule_interval)` relative to the
```

Comment:
Maybe it is easier to say that the most recent bucket lags behind at most `schedule_interval+end_interval` (given that there is data to materialize in that bucket).

Reply:
Reworded.